### PR TITLE
Terraform module references updated

### PR DESCRIPTION
**Pull request by russellseymour**
## What?

The dependent Terraform modules have been updated to include new outputs. The references to these modules has been updated.

## Why?

The dependent modules have been modified to provide more information about the base terraform state.

## How?

The modules are referenced using Git tags. This has been bumped to the latest version, `v1.5.1` of the `amido/stacks-terraform` module.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,7 @@
 .DS_Store
 
 .terraform
+.terraform.lock.hcl
+tfplan
+
+local
```
Comment (ElvenSpellmaker) [inline .gitignore:7]: newline please~

File: deploy/azure/infra/stacks-aks/outputs.tf [hunk 2]
```diff
@@ -82,7 +82,58 @@ output "issuer_pem" {
...
+output "app_insights_id" {
+  description = "ID of the Application Insights instance"
+  value = module.aks_bootstrap.app_insights_id
+}
+
+output "app_insights_key" {
+  description = "Shared key of the Application Insights instance"
+  value = module.aks_bootstrap.app_insights_key
+  sensitive = true
+}
```
Comment (ElvenSpellmaker) [inline deploy/azure/infra/stacks-aks/outputs.tf:139]: newline pls~

Comment (ElvenSpellmaker) [reply deploy/azure/infra/stacks-aks/outputs.tf:139]: If you use VSCode, you should set the editor to add them for you perhaps? 🙂

Comment (russellseymour) [reply deploy/azure/infra/stacks-aks/outputs.tf:139]: Fixed. Yep noted :-)

**Review by ElvenSpellmaker — APPROVED** [review]

**Pull request merged**